src/Servers/Kestrel/Transport.Quic/src/Microsoft.AspNetCore.Server.Kestrel.Transport.Quic.csproj

```diff
@@ -8,6 +8,7 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <NoWarn>CS1591;CS0436;$(NoWarn)</NoWarn><!-- Conflicts between internal and public Quic APIs -->
     <Nullable>enable</Nullable>
+    <RemoveDevicePlatformSupport>true</RemoveDevicePlatformSupport>
```

Comment:
Doesn't this just remove ios and android?

If we're going to make any changes in this area, I think we should look at what Runtime does and copy them.

https://github.com/dotnet/runtime/blob/c7e02f3773da25946328eba150a70caa3b0e4c74/eng/versioning.targets#L53-L85
and a usage example
https://github.com/dotnet/runtime/blob/d4818a32f449fae16a5edd3c056ade56471f51ee/src/libraries/System.Net.Quic/Directory.Build.props#L6

Reply:
Yea, I thought maybe that would be sufficient so I wanted to try it on the CI while I fix my machine. But looks like it's not sufficient.

Reply:
Related https://github.com/dotnet/aspnetcore-tooling/pull/3492#issuecomment-824289220